crates/subspace-farmer/src/bin/subspace-farmer/commands/cluster.rs

```diff
@@ -112,12 +114,14 @@ where
     .map_err(|error| anyhow!("Failed to connect to NATS server: {error}"))?;
     let mut registry = Registry::with_prefix("subspace_farmer");
 
-    let mut tasks = FuturesUnordered::new();
+    let mut tasks =
+        FuturesUnordered::<Pin<Box<dyn Future<Output = anyhow::Result<()>> + Send>>>::new();
```

Comment:
This type was previously guessed correctly by the compiler, but it has trouble guessing `+ Send` in `Box<dyn Future<Output = anyhow::Result<()>>`.